src/content/getting-started/architecture/globalnet/_index.en.md

```diff
@@ -78,10 +91,78 @@ Globalnet currently relies on `kube-proxy` and thus will only work with deployme
 Connectivity is only part of the solution as pods still need to know the IPs of services on remote clusters.
 
 This is achieved by enhancing [lighthouse](https://github.com/submariner-io/lighthouse) with support for Globalnet. The Lighthouse
-controller adds the service's globalIp to the `ServiceImport` object that is distributed to all clusters. The [lighthouse
-plugin](https://github.com/submariner-io/lighthouse/tree/devel/plugin/lighthouse) then uses the Service's globalIp when replying to DNS
-queries for the Service.
+controller uses pod's globalIp instead of podIp when creating `EndpointSlice` objects to be distributed to other clusters.
+The [lighthouse plugin](https://github.com/submariner-io/lighthouse/tree/devel/plugin/lighthouse) then uses the Service's globalIp when
+replying to DNS queries for the Service.
 
 ## Building
 
 Nothing extra needs to be done to build `submariner-globalnet` as it is built with the standard Submariner build.
+
+## Usage
+
+Refer [Quickstart Guides](../../quickstart/) on how to deploy Submariner with Globalnet enabled. For most deployments users will not
+need to do anything else once deployed. But users can create GlobalEgressIPs or edit ClusterGlobalEgressIP for specific use cases.
+
+### Ephemeral Port Exhaustion
+
+By default `8` globalIps are allocated as ClusterGlobalEgressIPs. This allows for `~8x64k` active ephemeral ports. If those are still
```

Comment:
```suggestion
By default 8 globalIps are allocated as ClusterGlobalEgressIPs. This allows for `~8x64k` active ephemeral ports. If those are still
```

Values like this shouldn't be code style:

> For field values of type string or integer, use normal style without quotation marks.

https://kubernetes.io/docs/contribute/style/style-guide/#use-normal-style-for-string-and-integer-field-values

Reply:
```suggestion
By default ,8 cluster-scoped global IPs are allocated which allows for `~8x64k` active ephemeral ports. If those are still
```